### PR TITLE
Draft of FN3 done!

### PR DESCRIPTION
**Pull request by tdegeorge**

### CODE REVIEW

File: source/02-FN/03.ptx [hunk 1]
```diff
@@ -11,6 +11,657 @@
...
+          <!--<image source="Images/Graph-with-two-endpoints.jpg"/>-->
+          <image>
+            <sageplot>
+              f(x) = 9/8*(x-4)+4
+              p=plot(f, (x, -4, 4), ymin=-7, ymax=7, xmin=-5,xmax=5,color='blue', thickness=3,)
+              p+=point((4,4),pointsize=50,color='blue')
+              p+=point((-4,-5),pointsize=50,color='blue')
+              p
+            </sageplot>
+          </image>
```
Comment (siwelwerd) [inline source/02-FN/03.ptx:166]: I think we should use `<sageplot>`s for these for consistency. I went ahead and did this one, but the others need to be done as well.

**Review by siwelwerd — CHANGES_REQUESTED** [review]
Aside from changing the images to `<sageplot>`s, I think this looks good.

File: source/02-FN/03.ptx (hunk 1 above)
Comment (StevenClontz) [reply source/02-FN/03.ptx:166]: I've added the others.

File: source/02-FN/03.ptx [hunk 2]
```diff
@@ -11,6 +11,692 @@
...
+        </p>
+        <figure xml:id="graph-of-ray-with-open-endpoint">
+          <!-- <image source="Images/Ray-with-open-endpoint.jpg"/> -->
+            <sageplot>
+              p=arrow((3,-4), (-4,8),xmin=-6,xmax=5,ymin=-6,ymax=8)
+              p+=point((3,-4),pointsize=50,color='blue')
+              p+=point((3,-4),pointsize=10,color='white',zorder=2)
+              p
+            </sageplot>
+        </figure>
```
Comment (siwelwerd) [inline source/02-FN/03.ptx:287]: Pretext throws a warning when `<sageplot>`s are not in an `<image>`.  Curious to see if the github action validates this.

Comment (siwelwerd) [reply source/02-FN/03.ptx]: Looks like it gave a green check, but I saw the warning there when I went into the details.  I like that I can see the full PreTeXt output there

Comment (StevenClontz) [reply source/02-FN/03.ptx]: yeah either the CLI should be erroring out there, or there should at least be a mode to run it that returns an error code. future work...

**Pull request merged**